t/t2802-uri-cmd.t

```diff
@@ -145,6 +145,22 @@ test_expect_success 'flux-uri mock testing of slurm resolver works' '
 	( export PATH=$(pwd):$PATH REMOTE=t SRUN_FAIL=t &&
 	  test_expect_code 1 flux uri slurm:1234 )
 '
+test_expect_success 'setup fake csm_allocation_query for mock lsf testing' '
+	cat <<-EOF >csm_allocation_query &&
+	#!/bin/sh
+	echo "compute_nodes:"
+	echo " - lassen9"
+	echo " - lassen10"
+	EOF
+	chmod +x csm_allocation_query
+	export CSM_ALLOCATION_QUERY=$(pwd)/csm_allocation_query
+	export FLUX_SSH=${SHARNESS_TEST_SRCDIR}/scripts/tssh
+'
+test_expect_success 'flux-uri mock testing of lsf resolver works' '
+	result=$(flux uri --local lsf:12345)
+	test "$result" = "$FLUX_URI" &&
+	test_expect_code 0 flux uri --local lsf:12345
+'
```

Comment:
Also consider adding a test for some failure cases, like invalid LSF jobid. This should raise the coverage of your code.